course_catalog/task_helpers.py

```diff
@@ -396,3 +404,25 @@ def generate_course_prefix_list(bucket):
             if "/".join(key_pieces[:-2]) != "":
                 ocw_courses.add("/".join(key_pieces[:-2]) + "/")
     return list(ocw_courses)
+
+
+def get_course_url(course_id, course_json, platform):
+    """
+    Get the url for a course if any
+    """
+    if platform == PlatformType.ocw.value:
+        if course_json is not None:
```

Comment:
This can be cleaned up a bit by putting a `if course_json is None: return None` at the top so we only perform that check once upfront.

Reply:
It's actually possible to  get a mitx url even if the JSON is None (just need the course id)

Reply:
Hmmm, right. Guess there's not much that can be improved in readability here...